rosa_learning/deploying_application_workshop/cloud-experts-deploying-application-networking.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-networking"]
+= Networking
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-networking
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-12-14
+
+The OSToy application uses intra-cluster networking to separate functions by using microservices.
+
+image::deploying-networking-arch.png[OSToy Diagram]
+
+In this workshop, there are at least two separate pods, each with its own service. One pod functions as the front end web application with a service and a publicly accessible route. The other pod functions as the backend microservice with a service object so that the front end pod can communicate with the microservice. 
+
+Communication occurs across the pods if there is more than one pod. Because of communication limits, the microservice is not accessible from outside the cluster and other namespaces or projects. The purpose of the microservice is to serve internal web requests and return a JSON object containing the current hostname (the pod's name) and a randomly generated color string. This color string displays a box with that color on the OSToy application web console.
```

Comment:
The sentence that starts with "Because" is a bit confusing to me, most likely b/c I do not understand the technical aspect of the topic...could you just say "...outside the cluster, other namespaces, or projects"?